fxa-auth-db-server/docs/API.md

```diff
@@ -62,34 +63,38 @@ The following datatypes are used throughout this document:
 * Devices:
     * createDevice              : `PUT /account/:id/device/:deviceId`
     * updateDevice              : `POST /account/:id/device/:deviceId/update`
-    * deleteDevice              : `DEL /account/:id/device/:deviceId`
+    * deleteDevice              : `DELETE /account/:id/device/:deviceId`
```

Comment:
If we're marking things up in `<code></code>` tags, they shouldn't be abbreviated imho. Semantic markup etc. Took the liberty of updating it while I was in here, since I'm adding another `DELETE` route.